src/sidebar.jsx

```diff
@@ -244,15 +244,6 @@ const DropdownWithKebab = ({
         ...selected.length === 0
             ? [{ id: "show-hidden-items", onClick: onToggleHidden, title: showHiddenItems }]
             : [],
-        {
-            id: "copy-path",
-            onClick: () => {
-                navigator.clipboard.writeText(path.join("/") + "/" + (selected.length === 1
```

Comment:
No veto, but the commit message is confusing. It says "*should be part of*", but is it actually? This should be definite, either "is", or "will be" if you are going to add it back later on.